README.md

```diff
@@ -129,13 +129,13 @@ cd data
 To create a passthrough pipeline, enter:
 
 ```
-scloud streams compile-dsl --input-datafile passthrough-dsl.json
+scloud streams compile --spl "$(< passthrough.spl )" > output.upl.json
```

Comment:
Just need to change the last line to `> passthrough.upl.json` to match the next command